alethzero/plugins/whisper/Whisper.cpp

```diff
@@ -113,10 +126,16 @@ Whisper::Whisper(ZeroFace* _m):
 {
 	dock(Qt::RightDockWidgetArea, "Whisper")->setWidget(new QWidget);
 	m_ui->setupUi(dock()->widget());
-	connect(addMenuItem("New Whisper Identity", "menuAccounts", true), &QAction::triggered, this, &Whisper::on_newIdentity_triggered);
+	connect(addMenuItem("New Whisper Identity", "menuAccounts", true), &QAction::triggered, this, &Whisper::onNewIdentityTriggered);
 	connect(zero()->web3Server(), &WebThreeServer::onNewId, this, &Whisper::addNewId);
-	connect(m_ui->post, SIGNAL(clicked()), this, SLOT(on_post_clicked()));
-	connect(m_ui->forget, SIGNAL(clicked()), this, SLOT(on_forget_clicked()));
+	connect(m_ui->post, SIGNAL(clicked()), this, SLOT(onPostClicked()));
+	connect(m_ui->forgetTopics, SIGNAL(clicked()), this, SLOT(onForgetTopicsClicked()));
+	connect(m_ui->forgetDestinations, SIGNAL(clicked()), this, SLOT(onForgetDestinationsClicked()));
+
+	QShortcut *shortcutSend1 = new QShortcut(QKeySequence("Ctrl+Enter"), m_ui->shhData);
```

Comment:
It is not a leak. Qt doc says (http://doc.qt.io/qt-5/objecttrees.html):
"When you create a QObject with another object as parent, it's added to the parent's children() list, and is deleted when the parent is. It turns out that this approach fits the needs of GUI objects very well. For example, a QShortcut (keyboard shortcut) is a child of the relevant window, so when the user closes that window, the shortcut is deleted too."